src/Build.UnitTests/Graph/GraphTestingUtilities.cs

```diff
@@ -17,20 +17,30 @@ internal static class GraphTestingUtilities
         public static readonly ImmutableDictionary<string, string> EmptyGlobalProperties = new Dictionary<string, string> {{PropertyNames.IsGraphBuild, "true"}}.ToImmutableDictionary();
 
         public static readonly string InnerBuildPropertyName = "InnerBuild";
+        public static readonly string InnerBuildPropertiesName = "InnerBuildProperties";
 
-        public static readonly string MultitargetingSpecification = $@"<PropertyGroup>
+        public static readonly string CrosstargetingSpecificationPropertyGroup = $@"<PropertyGroup>
```

Comment:
I thought we had moved everything to "multitargeting"? Why the change back?

Reply:
I thought multitargeting was changed to crosstargeting :) What's the official name?

Reply:
I read sometime back that multitargeting was the official name but kept crosstargeting in the msbuild property names like `IsCrossTargetingBuild` and `Custom*MicrosoftCommonCrossTargetingTargets` because of compat.

Reply:
Crosstargeting is the old name, supported for compat. But for new property names, call them Multitargeting.

Reply:
Alright, moved the naming back :)